crypto/keys/pem/pem_test.go

```diff
@@ -54,57 +53,6 @@ NHcCAQEEIHG5m/q2sUSa4P8pRZgYt3K0ESFSKp1qp15VjJhpLle4oAoGCCqGSM49AwEHoUQDQgAEvuyn
 `
 )
 
-func TestFromProto(t *testing.T) {
-	for _, test := range []struct {
-		desc     string
-		keyProto *keyspb.PEMKeyFile
-		wantErr  bool
-	}{
-		{
-			desc: "PEMKeyFile",
-			keyProto: &keyspb.PEMKeyFile{
-				Path:     "../../../testdata/log-rpc-server.privkey.pem",
```

Comment:
Still used?

Reply:
Unfortunately, yes, below in the same file, and in `crypto/keys/pem/proto/register_test.go` as well.